work/doc.go

```diff
@@ -18,5 +18,17 @@
 // This file is derived from miner/miner.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package work implements Klaytn block creation and mining.
+/*
+Package work implements Klaytn block creation and mining.
```

Comment:
How about adding `operations`?
```suggestion
Package work implements Klaytn block creation and mining operations.
```